FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/FlowcryptApiRepository.kt

```diff
@@ -136,4 +137,15 @@ class FlowcryptApiRepository : ApiRepository {
         apiService.getOpenIdConfiguration(url)
       }
     }
+
+  override suspend fun getPrivateKeysViaEkm(
+    context: Context,
+    ekmUrl: String,
+    tokenId: String
+  ): Result<EkmPrivateKeysResponse> =
+    withContext(Dispatchers.IO) {
+      val apiService = ApiHelper.getInstance(context).retrofit.create(ApiService::class.java)
+      val url = if (ekmUrl.endsWith("/")) ekmUrl else "$ekmUrl/"
```

Comment:
Excellent! Thank you for normalizing the url :+1: this caused a bug on browser extension and I came here to mention this - and it's already done :-)